contrib/javaee.wildfly/licenseinfo.xml

```diff
@@ -47,4 +44,23 @@
         <license ref="W3C2" />
         <comment type="CATEGORY_B" />
     </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/datasources_1_4.xsd</file>
+        <license ref="LGPL-2.1" />
```

Comment:
LGPL-2.1 is Category-X, i.e. we can't use/redistribute it. https://www.apache.org/legal/resolved.html#category-x

Reply:
this is a mistake it should be 1.4.1 which is EL1.0